src/Raven.Server/Config/Categories/StorageConfiguration.cs

```diff
@@ -20,8 +20,9 @@ public StorageConfiguration()
         [DefaultValue(DefaultValueSetInConstructor)]
         [ConfigurationEntry("Storage.DiscardVirtualMemory", ConfigurationEntryScope.ServerWideOrPerDatabase)]
         public bool DiscardVirtualMemory { get; set; }
-        
-        [Description("You can use this setting to specify a different path to temporary files. By default it is empty, which means that temporary files will be created at same location as data file.")]
+
+        [Description("Use this setting to customize the path for the temporary files of the 'Databases' and the 'System' directories. " + 
+                     "By default, these files are written to the same location as the Raven.voron data file under the 'Temp' folder in those directories.")]
```

Comment:
Why do you think that previous description was not good? IMO it was clear enough. It was generic so it applied to both - temp path of databases and system storage.

 The current one sounds more complex that I have doubts if it won't be confusing, in particular this part: "the temporary files of the 'Databases' and the 'System' directories". It sounds that 'Databases' directory have a common temporary files path - which is not the case.

Reply:
* Right - the "new" text is more complex, better not to use, 
especially now that I'm aware it also affects the `Configuration` temp files.

* The original motivation for the change was to point out that this key does Not affect other storage areas (e.g. indexes, backups)

* **Can it be something such as the below ? 
Or just close PR ?**   
```
Use this setting to customize the path for the temporary files of the System directory, 
each Database directory, and its related Configuration directory.
By default, the temporary files are created under the `Temp` folder in those directories.
```

Reply:
IMO the current description is generic and clear enough so I'm for just closing the PR